public/javascripts/sidebarSubtopics.js

```diff
@@ -31,13 +31,26 @@ $(document).ready(function(){
         this.classList.toggle("active");
 
         var panel = document.getElementById("subtopics");
-        if (panel.style.maxHeight){
+        var helpPanel = document.getElementById("help-panel");
+        if (panel.style.maxHeight) {
```

Comment:
Could you add comment(s) to this code to explain a bit more about what's happening?